### PR TITLE
Add Rainlender2-Pro.app

### PR DESCRIPTION
**Pull request by bdmorin**
##### Instructions
- Look for and complete the section relevant to your submission. Delete the others, including these `Instructions`.
- `{{cask_file}}` represents the cask file you’re submitting/editing (if applicable).
- If there’s a checkbox you can’t complete for any reason, that’s OK. Just explain in detail why you weren’t able to do so.
#### Editing an existing cask
- [x] Commit message includes cask’s name (and new version, if applicable).
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
#### Adding a new cask
- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
### Changes to the core
- [ ] Followed [hacking.md](https://github.com/caskroom/homebrew-cask/blob/master/doc/development/hacking.md).

### CODE REVIEW

**Comment by bdmorin** [comment]
Hmmm.. Everything passed when I tested. I'll fix.

File: Casks/rainlender.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'rainlender' do
```
Comment (jawshooah) [inline Casks/rainlender.rb]: Looks like this is misspelled. Please change the token to `rainlendar` and rename the file to `rainlendar.rb`

File: Casks/rainlender.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  version '2.13.1'
+  sha256 '22fc298423a0bc9c2e3ab8b0752ba353eed2cdd38695cda24d542b138ad511a9'
+
+  url "http://www.rainlendar.net/download/Rainlendar-Pro-#{version}.dmg"
+  name 'rainlender'
```
Comment (jawshooah) [inline Casks/rainlender.rb]: name 'Rainlendar Pro'

File: Casks/rainlender.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+  homepage 'http://www.rainlendar.net/'
+  license :commercial
+
+  app 'Rainlendar2-Pro.app'
```
Comment (jawshooah) [inline Casks/rainlender.rb]: `app "Rainlendar#{version.major}-Pro.app"`

**Comment by jawshooah** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 147f0b8311d768b170d9938e29862507df6f1711. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Pull request closed without merging**